node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerOperationsImpl.java

```diff
@@ -286,8 +276,6 @@ private static void addMounts(NodeAgentContext context, Docker.CreateContainerCo
                 context.pathInNodeUnderVespaHome("var/container-data"),
                 context.pathInNodeUnderVespaHome("var/db/vespa"),
                 context.pathInNodeUnderVespaHome("var/jdisc_container"),
-                context.pathInNodeUnderVespaHome("var/mediasearch"), // TODO: Remove when Vespa 6 is gone
-                context.pathInNodeUnderVespaHome("var/run"), // TODO: Remove - contains .pid files
```

Comment:
Not strictly related to Vespa 6, but was brought back to give time for customers to migrate off, which was a long time ago: https://github.com/vespa-engine/vespa/pull/10039. This directly has been deleted at every container start up since that time, so I dont think any one using it.